test/cloudant.connection.test.js

```diff
@@ -0,0 +1,24 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-connector-couchdb2
```

Comment:
`loopback-connector-couchdb2` -> `loopback-connector-cloudant`

Reply:
Missed that one, good catch